### ISSUE TITLE
Admin bar won't appear on front-end

### ISSUE THREAD

**Issue by jruck** [markjaquith/WordPress-Skeleton#46]
As long as I keep the site address as mydomain.com/wp, it appears. But once I change it to simply mydomain.com (and keep the WordPress Address as mydomain.com/wp), the Admin bar doesn't appear anymore on the front-end.

Themes that have a built-in gutter for when the admin bar should be displaying continue to show the gutter, just no admin bar. (See attached). I've tested this behavior in a copy of the Twenty Thirteen theme as well.

Anyone else getting this, or have any idea what might be going on?

![screen shot 2013-12-10 at 5 53 45 am](https://f.cloud.github.com/assets/2606818/1713386/83ba7d56-6189-11e3-9e8d-9062da79a86a.png)

**Comment by fahmad91** [comment]
@jruck Did you ever figure out why this was happening? I'm also running into the same issue.

**Comment by mathieuhays** [comment]
This issue might happen when you forget to include wp_head() and wp_footer() but it's not relevant to this repo.

**Comment by jruck** [comment]
Nope, but I think it was an issue w/ the theme I was using at the time (Satellite). Subsequent themes have not had this issue.

**Comment by fahmad91** [comment]
I just tried this with the latest twentyfifteen theme and I'm still getting the same issue. It seems to start happening as soon as I update the Site Address URL to remove the '/wp/'. 

![capture](https://cloud.githubusercontent.com/assets/1927187/8409852/a697a902-1e48-11e5-8547-0d020f51ab06.PNG)

**Comment by jruck** [comment]
Hm, works for me on some themes, but apparently, like this, not others.

**Comment by fahmad91** [comment]
Ok, I just cleared by cache and logged-out and logged-in again. 

It working fine now! :)
